File: stuff_usecases/src/main/java/android/montreal/devfest/stuff/usescases/DataToEntityMapper.java
package android.montreal.devfest.stuff.usescases;

import android.montreal.devfest.entities.RandomStuffEntity;

class DataToEntityMapper {
    static RandomStuffEntity map(RandomStuffData data) {
        if (data == null) {
            return null;
        }
        return new RandomStuffEntity(data.text, data.number);
    }
}

File: stuff_usecases/src/main/java/android/montreal/devfest/stuff/usecases/DataToEntityMapper.kt
package android.montreal.devfest.stuff.usecases

import android.montreal.devfest.stuff.entities.RandomStuffEntity

internal object DataToEntityMapper {
    fun map(data: RandomStuffData?): RandomStuffEntity? {
        return if (data == null) {
            null
        } else RandomStuffEntity(data.text, data.number)
    }
}
